src/style.css

```diff
@@ -0,0 +1,3 @@
+@import url(./reset.css);
+@import url(./stars.css);
+@import url(./card.css);
```

Comment:
It should be better card.css and stars.css to move to a new folder (for ex. blocks)